tests/distributions/test_transform.py

```diff
@@ -619,16 +631,26 @@ def test_transform_univariate_dist_logp_shape():
     assert m.logp(jacobian=True, sum=False)[0].type.shape == (4,)
 
 
-def test_univariate_transform_multivariate_dist_raises():
+@pytest.mark.parametrize(
+    argnames="jacobian", argvalues=[True, False], ids=["jacobian", "no_jacobian"]
+)
+def test_univariate_transform_multivariate_dist(jacobian):
     with pm.Model() as m:
         pm.Dirichlet("x", [1, 1, 1], transform=tr.log)
+    assert m.logp(jacobian=jacobian).type.shape == ()
```

Comment:
Let's test the logp is the expected one here manually

Reply:
I am a bit lost. What would be the assert statement ?
I know ` assert np.all(pm.logp(x, pt.log(test_value)).eval() < np.inf)` ✅ , but what is the expected relationship with `tr.LogTransform().log_jac_det(test_value).sum(-1)` ?